packages/agoric-cli/src/deploy.js

```diff
@@ -50,7 +49,10 @@ export default async function deployMain(progname, rawArgs, powers, opts) {
   const console = anylogger('agoric:deploy');
 
   const allowUnsafePlugins = opts.allowUnsafePlugins;
-  if (allowUnsafePlugins) {
+  const promptForUnsafePlugins = async () => {
```

Comment:
For functions this big, I'm glad you used a named function expression, rather than an IIFE. So no change suggested here. OTOH...